### ISSUE TITLE
In case of headword corrections, how do we generate hwnorm1 in local and cologne

### ISSUE THREAD

**Issue by drdhaval2785** [sanskrit-lexicon/csl-corrections#85]
Dear @funderburkjim ,
This question did not arise in the recent 7 changes I made, but it will be a frequent occurrence when a headword is also changed.
How and how often do we carry this change to hwnorm1c.txt and sanhw1.txt files in hwnorm1 repository.
Kindly append this note to the readme_howto.txt file please.

**Comment by funderburkjim** [comment]
Good observation.   The hwnorm1c list is currently only used in the simple-search display system.
See the instructions at https://github.com/sanskrit-lexicon/hwnorm1/blob/master/sanhw1/readme.txt.
I have copied the instructions to readme_howto.txt as requested.
This update is only required when there are changes to the headwords.  
I've also updated locally and at cologne, per the instructions.

Note:  While exploring this, I ran into unexpected behavior in simple search ('input=simple, input_simple=default, akarna.)
This is a separate problem which I need to investigate further.
